arkiverer/src/test/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/admin/AdminInterfaceTests.kt

```diff
@@ -33,8 +34,8 @@ import java.util.concurrent.TimeUnit
 @ActiveProfiles("test")
 @SpringBootTest
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
-@Import(ContainerizedKafka::class)
-class AdminInterfaceTests {
+//@Import(ContainerizedKafka::class)
```

Comment:
Remove uncommented code

Reply:
this is intermediate comit. it is not job done yet.